cmd/registry/cmd/controller-update.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"log"
+	"github.com/apigee/registry/cmd/registry/controller"
+    "github.com/spf13/cobra"
+)
+
+func init() {
+	controllerCmd.AddCommand(controllerUpdateCmd)
+	controllerUpdateCmd.Flags().String("manifest_path", "", "Path to manifest file.")
+	controllerUpdateCmd.MarkFlagRequired("manifest_path")
+}
+
+var controllerUpdateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Generate a list of commands to update the registry state (experimental)",
+	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+
+		flagset := cmd.LocalFlags()
+		manifestPath, err := flagset.GetString("manifest_path")
```

Comment:
If this is always required, could we make it a positional argument? I'd like to follow kubectl conventions for style; that suggests using FILENAME in the usage string. You can add that by changing 
```
Use: "update"
```
to 
```
Use: "update FILENAME"
```
on line 30 above.

Reply:
Done